modules/grpc_api/grpc_api.bal

```diff
@@ -0,0 +1,32 @@
+import ballerina/grpc;
+
+configurable int port = 9090;
+
+Album[] albums = [
+    {id: "1", title: "Blue Train", artist: "John Coltrane", price: 56.99},
+    {id: "2", title: "Jeru", artist: "Gerry Mulligan", price: 17.99},
+    {id: "3", title: "Sarah Vaughan and Clifford Brown", artist: "Sarah Vaughan", price: 39.99}
+];
+
+@grpc:Descriptor {
+    value: "DESCRIPTOR_RECORD_STORE"
```

Comment:
@daneshk I made this as a string to make this compile. I could not find the original constant value to compile?